README.md

```diff
@@ -112,100 +124,7 @@ $hashtableData = Read-CloudEventJsonData -CloudEvent $cloudEvent
 ```powershell
 $hashtableData = Read-CloudEventXmlData -CloudEvent $cloudEvent -ConvertMode SkipAttributes
 ```
-The `ConvertMode` parameter specifies how the XML to be represented in the result hashtable<br/>
-`SkipAttributes` - Skips attributes of the XML elements. XmlElement is a Key-Value pair where Key is the Xml element name, and the value is the Xml element inner text<br/>
-Example:
-```xml
-<key att='true'>value1</key>
-```
-is converted to
-```powershell
-@{'key' = 'value-1'}
-```
-`AlwaysAttrValue` - Each element is a HashTable with two keys<br/>
-        'Attributes' - key-value pair of the Xml element attributes if any, otherwise null<br/>
-        'Value' - string value represinting the xml element inner text<br/>
-Example:
-```xml
-```
-<key1 att='true'>value1</key1><key2>value2</key2>
-is converted to
-```powershell
-@{
-   'key1' = @{
-      'Attributes' = @{
-         'att' = 'true'
-      }
-      'Value' = 'value1'
-   }
-   'key2' = @{
-      'Attributes' = $null
-      'Value' = 'value2'
-   }
-}
-```
-`AttrValueWhenAttributes` - Uses `SkipAttributes` for xml elements without attributes and `AlwaysAttrValue` for xml elements with attributes<br/>
-Example:
-```xml
-<key1 att='true'>value1</key1><key2>value2</key2>
-```
-is converted to
-```powershell
-@{
-   'key1' = @{
-      'Attributes' = @{
-         'att' = 'true'
-      }
-      'Value' = 'value1'
-   }
-   'key2' = 'value2'
-}
-```
-
 ### Read CloudEvent Custom Format **Data** as a **byte[]**
 ```powershell
 $bytes = Read-CloudEventData -CloudEvent $cloudEvent
-```
-
-## Build the **CloudEvents.Sdk** Module
```

Comment:
Where does the build stuff go?

Reply:
I removed this section because the CONTRIBUTING.md covers it.

Reply:
👍